pkg/delete/internal/classic/delete.go

```diff
@@ -44,7 +44,7 @@ func Delete(ctx context.Context, client client.ConfigClient, dps []pointer.Delet
 		if theAPI.HasParent() {
 			parentID, e = resolveIdentifier(ctx, client, theAPI.Parent, toIdentifier(dp.Scope, "", ""))
 			if e != nil && !is404(e) {
-				log.WithFields(field.Error(e)).Error("unable to resolve config ID: %w")
+				log.WithFields(field.Error(e)).Error("unable to resolve config ID: %w", e)
```

Comment:
This will log the error twice

Reply:
for delete it is not the case. And I'm not going to expand the scope of this PR and fix it.

Reply:
```suggestion
				log.WithFields(field.Error(e)).Error("unable to resolve config ID: %s", e.Error())
```